docs/examples/terraform-and-ansible.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: default
+title: Terraform/Ansible Workflows
+nav_order: 10
+parent: Examples
+---
+
+# Using Terraform & Ansible in a Workflow
+
+Terraform and Ansible are both widely used technologies in the area of automating infrastructure deployments, configurations, and more.
+At first glance, it might seem daunting to incorporate these technologies into a Direktiv workflow; but it's actually fairly straightforward. 
+This article seeks to explain how to structure potentially 'complex' workflows in order to support technologies that may require data that persists across multiple workflow executions.
+
+## The Workflow
+
+This workflow consists of only 2 states. The first state runs the `terraform` isolate, and requires access to a variable that contains the contents of a `main.tf` file. This `main.tf` file includes all of the configuration and authorisation details required to create a new virtual machine on Google Cloud Platform. The state is configured to assign the public IP address of the resulting virtual machine as `.addr`
+
+The second state uses the `ansible` isolate, and requires access to two different variables:
+
+- `playbook.yml`
+  - Ansible playbook that will be executed to connect to the remote machine and print a 'Hello, world!' message.
+- `pk.pem`
+  - Private key included in the `main.tf` variable of the previous state, used by Ansible to securely connect to the remote virtual machine.
```

Comment:
This reads a little strange, I read this as your somehow getting a value out of main.tf . I would write it more like the `private key which matches the one in the main.tf variable` or `is the same as the one in main.tf`. Doesn't have to be those, but something with more clarity.